src/inc/msquic.h

```diff
@@ -295,6 +295,26 @@ typedef struct QUIC_LISTENER_STATISTICS {
     } Binding;
 } QUIC_LISTENER_STATISTICS;
 
+typedef enum QUIC_PERFORMANCE_COUNTERS {
+    QUIC_PERF_COUNTER_CONN_CREATED,
+    QUIC_PERF_COUNTER_CONN_HANDSHAKE_FAIL,
+    QUIC_PERF_COUNTER_CONN_APP_REJECT,
+    QUIC_PERF_COUNTER_CONN_RESUMED,
+    QUIC_PERF_COUNTER_CONN_ACTIVE,
+    QUIC_PERF_COUNTER_CONN_CLOSED_ERROR,
+    QUIC_PERF_COUNTER_CONN_NO_ALPN,
+    QUIC_PERF_COUNTER_PKTS_SUSPECTED_LOST,
+    QUIC_PERF_COUNTER_PKTS_DROPPED,
+    QUIC_PERF_COUNTER_PKTS_DECRYPTION_FAIL,
+    QUIC_PERF_COUNTER_DGRAM_RECV,
+    QUIC_PERF_COUNTER_DGRAM_SEND,
```

Comment:
I also want a send/recv event that increments by 1 for each send call or receive event.